doc/nrf/libraries/modem/pdn.rst

```diff
@@ -7,7 +7,7 @@ PDN
    :local:
    :depth: 2
 
-The PDN library can be used to manage Packet Data Protocol (PDP) contexts and PDN connections.
+The PDN library can be used to manage Packet Data Protocol (PDP) contexts and Packet Data Network (PDN) connections.
```

Comment:
We are having offline discussion on whether PDP context should be PDN context in the PDN library scope.

Reply:
Keeping PDP term for context.